src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -199,6 +199,16 @@ export class WizardModal extends Modal {
 		if (dialogPaneToShow && readHeader) {
 			status(`${dialogPaneToShow.pageNumberDisplayText} ${dialogPaneToShow.title}`);
 		}
+
+		// Remove the current page's custom buttons
+		this._wizard.pages[this._wizard.currentPage]?.customButtons.forEach(button => {
+			this.removeFooterButton(button.label);
```

Comment:
This looks like it's disposing the button - did you check to make sure that this isn't going to break anything (such as the onClick handlers)?

Reply:
yes, I've tested it, everything works fine